pykla/accounts/urls.py

```diff
@@ -0,0 +1,31 @@
+from django.urls import path
+from django.conf.urls import url
+from . import views
+from django.contrib.auth.views import LoginView,LogoutView, PasswordResetView, PasswordResetDoneView, \
+    PasswordResetConfirmView, PasswordResetCompleteView
+
+urlpatterns=[
```

Comment:
@Dru94 what do you think about using social login instead, I'm thinking maybe GitHub and Google. I suggest you use [Django Allauth](https://django-allauth.readthedocs.io/en/latest/) but you can suggest any other package that does the same thing